frontend/src/metabase/selectors/app.ts

```diff
@@ -85,9 +80,7 @@ export const getIsNavBarEnabled = createSelector(
     if (isEmbedded && !embedOptions.side_nav) {
       return false;
     }
-    if (isEmbedded && embedOptions.side_nav === "default") {
-      return EMBEDDED_PATHS_WITH_NAVBAR.some(pattern => pattern.test(path));
-    }
```

Comment:
this led to a sort-of broken feeling behavior where you could not expand the sidebar on question/dashboard/model pages, but it would show up on the home + collection pages.

Deleting this block gives us the behavior we get in non-embedded contexts where the sidebar is hidden by default on questions/models/dashboards, but can be expanded at the user's desire.